src/main/java/frc/robot/Constants.java

```diff
@@ -49,25 +51,33 @@
 
         public static final double STOP_CLIMBER = 149.5; // [s]
```

Comment:
Add "timestamp" or something similar to this constant's name. It's not clear whether this is a time diff or a target angle.